clang/test/CheckedCRewriter/fptrarrstructprotocaller.c

```diff
@@ -176,13 +178,15 @@ return z; }
 //CHECK_NOALL:         _Ptr<struct fptrarr> y =   malloc(sizeof(struct fptrarr));
 //CHECK_NOALL:         int *yvals = calloc(5, sizeof(int)); 
 //CHECK_NOALL:         for(int i = 0; i < 5; i++) {
+//CHECK_NOALL:         strcpy(y->name, ((const char *)"Example")); 
 //CHECK_NOALL:         struct fptrarr *z = sus(x, y);
 //CHECK_ALL: struct fptrarr * bar() {
 //CHECK_ALL:         char name[20]; 
 //CHECK_ALL:         struct fptrarr * x = malloc(sizeof(struct fptrarr));
 //CHECK_ALL:         _Ptr<struct fptrarr> y =   malloc(sizeof(struct fptrarr));
-//CHECK_ALL:         int *yvals = calloc(5, sizeof(int)); 
+//CHECK_ALL:         _Array_ptr<int> yvals: count((5 * sizeof(int))) =  calloc(5, sizeof(int)); 
 //CHECK_ALL:         for(int i = 0; i < 5; i++) {
+//CHECK_ALL:         strcpy(y->name, ((const char *)"Example")); 
```

Comment:
Any idea why this is here now, but wasn't before?

Reply:
Yes, this is a bug in the testgen script. When ID-ing what to annotate, I have a set of keywords (int, char, _Ptr, ....) that it looks for. The idea here is to annotate both `_Ptr<char>` and `char *` wherever they appear in the converted code. I can fix these errant tests and push up again. What's weird is that I made the changes to the `testgenerator` script way back during the `malloc fix` PR so it is weird that they changed now. At any rate, I'll fix my script and see if this problem vanishes.  

Also, quick Q, what is the "small refactoring for the code" change you requested? I only see the comments you made on the two tests.

Reply:
Correction, not "fix these errant tests", should be "fix my script"